packages/flutter_tools/pubspec.yaml

```diff
@@ -121,11 +103,26 @@ dev_dependencies:
   build_vm_compilers: 1.0.4
   build_test: 0.10.12+1
 
+  bazel_worker: 0.1.23+1 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
A lot of these libraries are still depended on via build_test, but that might be removable in the future